tests/UtilsTest.php

```diff
@@ -916,4 +916,26 @@ public static function dataValidClassAndMethodPair() {
 			[ [ 'Exception', 'getMessage' ], true ],
 		];
 	}
+
+	public function testGetSizeStringFromBytes() {
+		$test_cases = [
+			[ 0, 2, '', '0 B' ],
+			[ 500, 2, '', '500 B' ],
+			[ 1000, 2, '', '1 KB' ],
+			[ 1500, 2, '', '1.5 KB' ],
+			[ 1000000, 2, '', '1 MB' ],
+			[ 1000000000, 2, '', '1 GB' ],
+			[ 1000000000000, 2, '', '1 TB' ],
+			[ 1000, 0, 'KB', '1 KB' ],
+			[ 1000000, 0, 'MB', '1 MB' ],
+			[ 1000000000, 0, 'GB', '1 GB' ],
+			[ 1000000000000, 0, 'TB', '1 TB' ],
```

Comment:
These test scenarios are not really testing the decimals properly. Please include non-rounded numbers as well, to make sure the decimals cut off at the right place.

Reply:
@schlessera 
Thanks for the review.

I've added tests for non-rounded numbers.

> make sure the decimals cut off at the right place

Can you clarify on this a bit.
For test case: `get_size_string_from_bytes( 1500, 2, '')` should it return to `1.5 KB` or `1.50 KB`?
Currently it returns `1.5 KB` removing the extra zero at the end.